includes/class-wc-payments-payment-request-button-handler.php

```diff
@@ -373,14 +373,15 @@ public function add_order_meta( $order_id, $posted_data ) {
 
 		$payment_request_type = wc_clean( wp_unslash( $_POST['payment_request_type'] ) ); // phpcs:ignore WordPress.Security.NonceVerification
 
+		$suffix = apply_filters( 'wcpay_payment_request_payment_method_title_suffix', ' (WooCommerce Payments' );
```

Comment:
Should there be a ending parentheses after `(WooCommerce Payments` ?

Reply:
I've tested it with Google Pay and it works, but I've got "WooCommerce PaymentsTaha Paksu" when I only returned "Taha Paksu" from the filter. I wonder if we should also force the ` (` and `)` for the string returned, so they could only replace what's inside the parentheses? What do you think? Not required though.

Reply:
I personally prefer your idea, @tpaksu. However, going back to [the original request](https://github.com/Automattic/woocommerce-payments/pull/2435#issuecomment-992685127), I think the intention is to remove this suffix string entirely. In this case, using the filter to submit an empty string would leave empty parentheses (`()`) at the end of the payment method title. For more reference, the changes here follow the [format on the Stripe payment gateway](https://github.com/woocommerce/woocommerce-gateway-stripe/blob/44f8f59a98b0fe8502008c28be8ab087e866723d/includes/payment-methods/class-wc-stripe-payment-request.php#L83).

Reply:
Forgetting the closing parenthesis was a goof on my part though! I added it in as of 4b315b2321181aa48d69653b6918abb191ea19ac.

Reply:
> Or better, if they send an empty string, we can completely remove the (), otherwise keep the parentheses.

Whoops, I didn't see this comment: followed your sage advice to the letter in 66eda5e7d6df3e1a47028db54cbed14ede50c001.